scripts/utils.py

```diff
@@ -69,7 +69,7 @@
 GAS_PARAMS = {
     "maxFeePerGas": 30_000_000_000,
     "maxPriorityFeePerGas": 3_000_000_000,
-    "gas": 500_000,
+    "gas": 5_000_000,
```

Comment:
Verify

Reply:
@0xArdi `"gas": 500_000` has caused issues sometimes on Tenderly when sending the unstake transaction. I suspect it usually runs out of gas due to being executed when several checkpoints have not been called (thus it needs to do extra work). This seems not happening on mainnet, where the checkpoint is periodically called. What do you suggest to do here?

Reply:
Ideally we do this using automatic gas estimation.

Reply:
Yes, but in this script we are not doing it. So what should we do? keep it asi it is 500.000 ? increase it?